workflows/computational-chemistry/gromacs-dctmd/gromacs-dctmd.ga

```diff
@@ -0,0 +1,2488 @@
+            "tool_version": null,
+            "type": "parameter_input",
+            "uuid": "a9d402ec-eafb-4fab-9e5a-192765011bba",
+            "workflow_outputs": []
+        },
+        "3": {
+            "annotation": "MD step length",
+            "content_id": null,
+            "errors": null,
+            "id": 3,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "MD step length",
+                    "name": "Step length (ps)"
+                }
+            ],
+            "label": "Step length (ps)",
+            "name": "Input parameter",
+            "outputs": [],
+            "position": {
+                "bottom": 787.6999969482422,
+                "height": 82.19999694824219,
+                "left": -839.5,
+                "right": -639.5,
+                "top": 705.5,
+                "width": 200,
+                "x": -839.5,
+                "y": 705.5
+            },
+            "tool_id": null,
+            "tool_state": "{\"parameter_type\": \"float\", \"optional\": false}",
+            "tool_version": null,
+            "type": "parameter_input",
+            "uuid": "bbc28f7f-ac02-42af-ad8b-e5d4137d348b",
+            "workflow_outputs": []
+        },
+        "4": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/get_online_data/ctb_online_data_fetch/0.4",
+            "errors": null,
+            "id": 4,
+            "input_connections": {},
+            "inputs": [],
+            "label": null,
+            "name": "Online data",
+            "outputs": [
+                {
+                    "name": "output",
+                    "type": "txt"
+                }
+            ],
+            "position": {
+                "bottom": 776.3000030517578,
+                "height": 61.80000305175781,
+                "left": -1117.5,
+                "right": -917.5,
+                "top": 714.5,
+                "width": 200,
+                "x": -1117.5,
+                "y": 714.5
+            },
+            "post_job_actions": {
+                "HideDatasetActionoutput": {
+                    "action_arguments": {},
+                    "action_type": "HideDatasetAction",
+                    "output_name": "output"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/get_online_data/ctb_online_data_fetch/0.4",
+            "tool_shed_repository": {
+                "changeset_revision": "2538366eb8fb",
+                "name": "get_online_data",
+                "owner": "bgruening",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"url_paste\": \"\https://raw.githubusercontent.com/simonbray/iwc/add-md-wfs/workflows/computational-chemistry/gromacs-dctmd/tmd.mdp", \"whitelist\": \"mdp\", \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
This 'trick' allows a fixed config file to be specified, analogously to `<config>` tags in tool wrappers. I hope this is acceptable. I'm not sure where the best place to put it is - it would be nice to package it with the workflow, if possible.